koku/api/report/view.py

```diff
@@ -22,21 +23,22 @@
 LOG = logging.getLogger(__name__)
 
 
-def get_paginator(filter_query_params, count, group_by_params=False):
+def get_paginator(filter_query_params, count, group_by_params=False, **kwargs):
```

Comment:
instead of kwargs, let's just accept a report_type argument with a default empty string:
```
def get_paginator(filter_query_params, count, group_by_params=False, report_type=""):
```